operations/configure_cray_cli.md

```diff
@@ -21,50 +25,195 @@ The `cray` CLI only needs to be initialized once per user on a node.
+   2021-12-21 15:52:32,714 - INFO    - ncn-w003: Success
+   2021-12-21 15:52:33,079 - INFO    - Re-initializing the cray cli with existing keycloak user MY_USERNAME
+   2021-12-21 15:52:33,131 - INFO    - Initializing nodes:
+   2021-12-21 15:52:37,714 - INFO    - ncn-m001: Success
+   2021-12-21 15:52:37,714 - INFO    - ncn-m002: Success
+   2021-12-21 15:52:37,714 - INFO    - ncn-m003: Success
+   2021-12-21 15:52:37,714 - INFO    - ncn-s001: Success
+   2021-12-21 15:52:38,714 - INFO    - ncn-s002: Success
+   2021-12-21 15:52:38,714 - INFO    - ncn-s003: Success
+   2021-12-21 15:52:38,714 - INFO    - ncn-w001: Success
+   2021-12-21 15:52:38,714 - INFO    - ncn-w002: Success
+   2021-12-21 15:52:38,714 - INFO    - ncn-w003: Success
    ```
+
+   At this point the `cray` CLI will be operational on all successful nodes and authenticated with
+   the input keycloak account.
+
+1. Initialize the `cray` CLI on a single node with an existing keycloak user account.
+
+   To initialize the `cray` CLI on a single node with an existing keycloak account use the
+   `cray init` command with the correct API Gateway hostname and keycloak account and password.
+   Expected output should look something like the following:
+   ```bash
+   ncn# cray init
    Cray Hostname: api-gw-service-nmn.local
-   Username: vers
-   Password:
+   Username: MY_KEYCLOAK_USER_NAME
+   Password: MY_PASSWORD
    Success!
 
    Initialization complete.
    ```
 
-1. Verify the `cray` CLI is operational.
-    ```bash
-    ncn# cray artifacts buckets list -vvv
-    ```
-
-    Expected output, if an error occurs see the troubleshooting section below in this topic.
-    ```
-    Loaded token: /root/.config/cray/tokens/api_gw_service_nmn_local.vers
-    REQUEST: PUT to https://api-gw-service-nmn.local/apis/sts/token
-    OPTIONS: {'verify': False}
-    S3 credentials retrieved successfully
-    results = [ "alc", "badger", "benji-backups", "boot-images", "etcd-backup", "fw-update", "ims", "install-artifacts", "nmd", "postgres-backup",
-    "prs", "sat", "sds", "sls", "sma", "ssd", "ssm", "vbis", "velero", "wlm",]
-    ```
+   The `cray` CLI may need to be authenticated to complete the setup. With the same keycloak username
+   and password that was used for initialization above, do the following:
+   ```bash
+   ncn# cray auth login
+   Username: MY_KEYCLOAK_USER_NAME
+   Password: MY_PASSWORD
+   Success!
+   ```
 
 ## Troubleshooting
 
+   Each node will have `Success` reported if everything worked and the node was initialized
+   and the `cray` CLI is operational for that node. For nodes with problems, there will be a
+   brief error message that reports what the problem is on that node.
+
+   ```bash
+   ncn# python3 /usr/share/doc/csm/install/scripts/craycli_init.py --run
+   2021-12-21 15:50:47,814 - INFO    - Loading keycloak secrets.
+   2021-12-21 15:50:48,095 - INFO    - Created user 'craycli_tmp_user'
+   2021-12-21 15:50:52,714 - INFO    - Initiailizing nodes:
```

Comment:
Should be Initializing, not Initiailizing. I see you already fixed this in the script itself, so just need to fix it in the example output.